### PR TITLE
Change the params of embeddings.py

### PR DESCRIPTION
**Pull request by DirkFi**
Change the params to accept both `embeddings_regularizer` and `activity_regularizer`. When `embeddings_regularizer` is not `None`, the user will get a warning.

### CODE REVIEW

File: python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py [hunk 1]
```diff
@@ -63,13 +68,36 @@ def __init__(self,
...
-        super().__init__(input_dim=input_dim,
-                         output_dim=output_dim,
-                         embeddings_initializer=embeddings_initializer,
-                         embeddings_regularizer=None,
-                         embeddings_constraint=embeddings_constraint,
-                         activity_regularizer=regularizer,
-                         mask_zero=mask_zero,
-                         input_length=input_length,
-                         **kwargs)
+        if embeddings_regularizer is None:
```
Comment (yangw1234) [inline python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py:71]: I think the changes could be simpler.

```python
if embeddings_regularizer is not None and activity_regularizer is None:
    warnings.warn('Apply regularizer directly on embeddings will make the sparse gradient dense and may result in degraded performance. We are changing your regularizer to apply on the output tensors ', UserWarning)
    activity_regularizer = embeddings_regularizer
    embeddings_regularizer = None

super().__init__(input_dim=input_dim,
                         output_dim=output_dim,
                         embeddings_initializer=embeddings_initializer,
                         embeddings_regularizer=embeddings_regularizer,
                         activity_regularizer=activity_regularizer,
                         embeddings_constraint=embeddings_constraint,
                         mask_zero=mask_zero,
                         input_length=input_length,
                         **kwargs)


```

File: python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py [hunk 2]
```diff
@@ -45,7 +47,10 @@ def __init__(self,
         :param output_dim: Integer. Dimension of the dense embedding.
         :param embeddings_initializer: Initializer for the `embeddings`
             matrix (see `keras.initializers`).
-        :param regularizer: Regularizer function applied to
+        :param embeddings_regularizer: Regularizer function applied to
+            the `embeddings` matrix (see `keras.regularizers`).
+            We recommend you not using this parameter.
```
Comment (yangw1234) [inline python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py:52]: "Applying regularizer directly on embeddings will make the sparse gradient dense and may result in degraded performance. We recommend you to use activity_regularizer."

File: python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py (hunk 1 above)
Comment (DirkFi) [reply python/nano/src/bigdl/nano/tf/keras/layers/embeddings.py:71]: ok

**Comment by DirkFi** [comment]
The successful PR validation is here [#214](http://10.112.231.51:18888/view/BigDL-2.0-PR/job/ZOO-PR-NanoTests/214/).

File: python/nano/test/tf/keras/test_embeddings.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ def test_embedding_gradient_sparse():
     from bigdl.nano.tf.keras import Sequential
     import tensorflow as tf
     model = Sequential()
-    model.add(Embedding(1000, 64, input_length=10, regularizer=tf.keras.regularizers.L2()))
+    model.add(Embedding(1000, 64, input_length=10, activity_regularizer=tf.keras.regularizers.L2()))
```
Comment (yangw1234) [inline python/nano/test/tf/keras/test_embeddings.py:27]: I think you should test `embeddings_regularizer` here, otherwise, it won't run into your updated logic.

Comment (DirkFi) [reply python/nano/test/tf/keras/test_embeddings.py:27]: Ok.

**Comment by DirkFi** [comment]
The latest pr validation is here [#216](http://10.112.231.51:18888/view/BigDL-2.0-PR/job/ZOO-PR-NanoTests/216/). In [result console](http://10.112.231.51:18888/job/ZOO-PR-nano-tensorflow-test/46/console), the `Userwarning` we add in function is called, which is expected.

**Comment by yangw1234** [comment]
LGTM
